src/lineContent.js

```diff
@@ -0,0 +1,29 @@
+import Router from 'koa-router';
+import { downloadFile } from './webhook/handlers/fileHandler';
+import { verify, read } from 'src/lib/jwt';
+
+const lineContentRouter = Router();
+
+lineContentRouter.get('/', async ctx => {
+  const jwt = ctx.query.token;
+  if (!jwt || !verify(jwt)) {
+    const err = new Error('`token` is invalid or expired.');
+    err.status = 400;
+    err.expose = true;
+    throw err;
+  }
+
+  const parsed = read(jwt);
+  const result = await downloadFile(parsed.messageId);
```

Comment:
This will download the whole file from LINE to chatbot before sending to response. Could we directly pipe body to response instead? In this way the file will not fill up the memory of chatbot.

Also, it would be great if this proxy can forward `Content-Type` and `Content-Length` from LINE `getContent` endpoint, as media-manager would rely on these two headers to determine file type and process files.